apps/src/templates/courseOverview/CourseScript.js

```diff
@@ -86,6 +86,7 @@ class CourseScript extends Component {
           ...(isHidden && styles.hidden)
         }}
         className="uitest-CourseScript"
+        data-visibility={isHidden ? 'hidden' : 'visible'}
```

Comment:
I added this marker because the style differences on the Hidden/Visible buttons between their selected/unselected states seem too subtle to rely on in UI tests, and for some reason the `$(':disabled')` selector wasn't working for me.